resources/js/components/contents/Dashboard.js

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
```

Comment:
You can add here the `useState` para no need to add `React.` as prefix when using `useState`

```suggestion
import React, { useState } from 'react';
```

Reply:
I have remove the the contents folder and all it's files including the dashboard.js . Sorry for the inconvenience ma'am.